src/realm/obj.cpp

```diff
@@ -39,6 +40,35 @@
 
 namespace realm {
 
+ConstObj ObjLink::get_obj(const Group& g) const
```

Comment:
I  think it would be more logical to have a method on Group that would translate an ObjLink into an Obj.

Reply:
Of course.